lib/mock-nexmo.js

```diff
@@ -54,7 +54,8 @@ function MockNexmo(log, config) {
 
         // HACK: Enable remote tests to see what was sent
         mailer.sendMail({
-          from: `sms.${senderId}@restmail.net`,
+          from: config.smtp.sender,
+          sender: config.smtp.sender,
```

Comment:
Do we need both properties or is one of them sufficient? Seems funny to duplicate it...